src/helpers/dataParser.helper.ts

```diff
@@ -0,0 +1,77 @@
+import { format, formatISO, parseISO } from "date-fns";
+import { ColumnParsingMap, DataTypesToUse } from "../types/excel.types";
+import { Table } from "../types/table.types";
+
+enum SupportedDataTypes {
+    date = "(DATE)",
+    number = "(NUMBER)"
+}
+
+enum DataTypes {
+    string = 'string',
+    number = 'number',
+    date = 'date'
+}
+
+const parseDateISO = (date: string) => {
+    try {
+        return date ? formatISO(parseISO(date)) : null;
+    } catch {
+        return null;
+    }
+};
+
+const formatDate = (date: string) => {
+    try {
+        return date ? format(parseISO(date), 'dd/MM/yyyy') : null;
+    } catch {
+        return null;
+    }
+};
+
+const parseString = (value: any) => value ? value.toString() : null;
+const parseNumber = (value: any) => value ? parseInt(value) : null;
+
+export const ParserToSaveMap: Record<DataTypes, (value: any) => DataTypesToUse> = {
+    string: parseString,
+    date: parseDateISO,
+    number: parseNumber
+};
+
+export const ParserToSendMap: Record<DataTypes, (value: any) => DataTypesToUse> = {
```

Comment:
and this is to send back to the client rite?

Reply:
yes I'll add comments